src/how-to/render/display-text.md

```diff
@@ -45,15 +45,17 @@ fn ui(_app: &App, f: &mut Frame<'_>) {
 }
 ```
 
-`Span`s are the building blocks to build styled content that is text that is represented
-horizontally.
+A `Span` is the basic building block for any styled text, and can be used anywhere text is
+displayed.
 
 ## `Line`
 
-The next building block that we are going to talk about is a `Line`. A `Line` represents a single
-"line" of text. The `Line` struct is essentially a `Vec<Span>`.
+The next building block that we are going to talk about is a `Line`. A `Line` represents a cluster
+of graphemes, where each unit in the cluster can have its own style. You can think of an instance of
+the`Line` struct as essentially a collection of `Span` objects, i.e. `Vec<Span>`.
```

Comment:
It needs a space after "the" and before `Line`

Reply:
Resolved in https://github.com/ratatui-org/ratatui-book/pull/72/commits/1bca04aa1fa3fff381885f8fc7208e3f836d5da5